.github/workflows/dependabot-labeler.yaml

```diff
@@ -1,10 +1,10 @@
 # This workflow file adds the 'lgtm' and 'approved' labels to Dependabot PRs
-# This is done to ensure that the PRs that pass e2e are automatically merged/added to merge-queues by the CodeFlare bot
+# This is done to ensure that the PRs that pass required status checks are automatically merged by the CodeFlare bot
 name: Dependabot Labeler
 
 on:
-  pull_request:
-    types: [ labeled, synchronize, opened, reopened ]
+  pull_request_target:
```

Comment:
what's the difference between `pull_request` and `pull_request_target`?

Reply:
Previously the workflow wouldn't work because the token was not recognized. When a user (in this case, the Dependabot actor) opens a PR from a fork, this fork has no access to the required secret to run the workflow. This is a GitHub security feature. To enable usage of the secret from a PR from a fork, I needed to run the workflow on `pull_request_target`, this way providing access to the secret which has very minimal permissions.